lithops/executors.py

```diff
@@ -511,6 +515,41 @@ def get_result(self, fs=None, throw_except=True, timeout=None,
 
         return result
 
+    def job_summary(self, cloud_objects_n=0):
+        """
+        logs information of a job executed by the calling function executor.
+        currently supports: code_engine, ibm_vpc and ibm_cf.
+
+        :param cloud_objects_n: number of cloud object used by user in COS.
+        """
+
+        def append(content):
+            """ appends information processed to a log file."""
+            headers = ['Function Name', 'Invocations', 'Memory Used(MB)',
+                       'Average Runtime(sec)', 'Cost($)', 'Cloud Objects Used']
+            pd.DataFrame(content).to_csv(path, mode='a', header=headers, index=False)
+
+        def calc_cost(total_runtime, memory_gb):
+            """ returns total cost associated with executing the calling function-executor's job."""
+            return self.config['unit_price'] * memory_gb * total_runtime
```

Comment:
I think the `calc_cos()` function should be implemented in the backend itself, since each backend has its own way to calculate the costs. For example, this function now works for IBM CF, but for CE or VPC it is also needed to take into account the amount of CPU.